autoreject/utils.py

```diff
@@ -2,12 +2,40 @@
 
 # Authors: Mainak Jas <mainak.jas@telecom-paristech.fr>
 
+import numpy as np
 import mne
+from mne.utils import logger
```

Comment:
@dengemann we can't use logger from another package. There is no control. Might as well use a print statement